tests/test_runner/test_hooks.py

```diff
@@ -316,6 +316,52 @@ def test_one_cycle_runner_hook():
     ]
     hook.writer.add_scalars.assert_has_calls(calls, any_order=True)
 
+    # Test OneCycleLrUpdaterHook
```

Comment:
Could we use `pytest.mark.parametrize` to remove redundant code like https://github.com/open-mmlab/mmcv/blob/master/tests/test_runner/test_hooks.py#L398